SnapWar/ViewControllers/SWMainNavViewController.swift

```diff
@@ -8,6 +8,13 @@ class SWMainNavViewController: UITabBarController {
         setup()
     }
     
+    override func viewDidAppear(_ animated: Bool) {
+        if(SWUserEntity.shared.externalId == "") {
```

Comment:
a helper method would be useful, such as `func isLoggedIn()`. we can just have a private boolean variable in the entity that indicates if the user is logged in or not, and this function can just return that. this variable is only set to true in `fromLoginUserInfo()` method